projects/cli/src/effects.js

```diff
@@ -38,6 +38,13 @@ const mode = () =>
     ? 'production'
     : 'development'
 
+const maybeViteConfig = async () => {
+  const maybeViteConfig = path.join(process.cwd(), 'vite.configs.js')
```

Comment:
This might be a minor typo 👇 

```suggestion
  const maybeViteConfig = path.join(process.cwd(), 'vite.config.js')
```

Reply:
Ah yes, I'll fix it. Also, proxy passing seems not to be working (it's detected) but not working. Unsure if it's related to how the server is started.